subprojects/build.gradle.kts

```diff
@@ -287,7 +287,7 @@ subprojects {
 
                 systemProperty("isTest", true)
 
-                systemProperty("junit.jupiter.execution.timeout.default", MINUTES.toSeconds(5))
+                systemProperty("junit.jupiter.execution.timeout.default", MINUTES.toSeconds(10))
```

Comment:
Will you plan investigation?

Reply:
created internal MBS-10204 for this

Reply:
Where did you see the statistics?

Reply:
it's in task description, i don't have one, just seeing this time to time on Teamcity's "Test" tab

Reply:
How many tests require 10 min? Can we override timeout only for them?

Reply:
i just don't know; seems like it's random with GradleTestKit one's
we didn't have any of timeouts, now added 10 min timeout and planned to add some statistic and fix it
i see it as an improvement worth merging